src/components/SummaryConfirmation/index.js

```diff
@@ -65,7 +123,7 @@ const SummaryConfirmation = ({submissionAllowed, onPrevPage}) => {
               variant="primary"
               name="confirm"
               disabled={loading || submitDisabled}
-              onDisabledClick={() => !loading && setWarningUncheckedPrivacy(true)}
+              onDisabledClick={() => !loading && showWarnings(formikValues)}
```

Comment:
looking at this and the code above, I think this should still be boolean state? Either warnings should be shown or not. However, the single checkbox requires some additional display logic:

1. StatementCheckbox takes props `label`, `name` and `showWarning`
2. each checkbox can use `useFormikContext().getFieldProps(name).value` to check whether it's checked or not
3. only display the warning if `showWarning && !value` (so don't display the warning if it's checked and if it's unchecked only display when the disabled submit button was clicked)

Clicking a checkbox to enable it will then make the warning disappear for that particular field. The `setWarning` would then be called if all checkbox fields are checked.